src/oidcop/user_authn/authn_context.py

```diff
@@ -108,58 +108,45 @@ def default(self):
             return None
 
 
-def pick_auth(endpoint_context, areq, all=False):
+def pick_auth(endpoint_context, areq, pick_all=False):
```

Comment:
all is a python primitive, better to avoid to overwrite it even if in this frame only